include/deal.II/grid/reference_cell.h

```diff
@@ -193,7 +190,7 @@ namespace ReferenceCell
 
 
 
-  inline Type::Type(const CellKinds kind)
+  inline Type::Type(const std::uint8_t kind)
     : kind(kind)
   {}
```

Comment:
If we aren't going to take an enum we should do a range check here to make sure we construct a valid object.

Reply:
I have a follow-up patch in which this will disappear. Just give me a day or two once this PR is merged.